libcontainer/configs/cgroup.go

```diff
@@ -57,6 +57,11 @@ type Cgroup struct {
 	// MEM to use
 	CpusetMems string `json:"cpuset_mems"`
 
+	// Process limit; set to `0' to disable limit.
+	// While technically `0' is a valid PID limit, it does not make sense in the
+	// context of a container -- it is identical to a limit of `1'.
```

Comment:
> And I think pids cgroup is a supporter for opencontainers/specs#233 .

@yangdongsheng I don't understand why you would think that. You can't set a value of `-1` to mean `max` in the PIDs controller ...?

I also don't agree with opencontainers/specs#233 (I just commented with my reservations). I'd be okay with it if we change all of the limits to be pointers (so you can set `null` to mean default, or omit the option to also mean default) and then get consumers to deal with the default values explicitly (by checking for null).